src/mcts/search.cc

```diff
@@ -124,6 +127,9 @@ void Search::SendUciInfo() REQUIRES(nodes_mutex_) {
     ++multipv;
     uci_infos.emplace_back(common_info);
     auto& uci_info = uci_infos.back();
+  if(params_.GetMultiPv() > 1 && params_.GetMultiPVDisaggregatedNodecount()) {
```

Comment:
`GetMultiPVDisaggregatedNodecount` is pretty slow to call in a loop, better to have:
```cpp
const bool multipv_counters = params_.GetMultiPv() > 1 && params.GetMultiPvCounters();
```
and then use it.